server/sonar-db-dao/src/main/java/org/sonar/core/permission/GlobalPermissions.java

```diff
@@ -26,7 +26,9 @@
 /**
  * Holds the constants representing the various global permissions that can be assigned to users & groups
  *
+ * @deprecated replaced by enum {@link org.sonar.db.permission.OrganizationPermission}
  */
+@Deprecated
 public final class GlobalPermissions {
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1133)